entrypoint.sh

```diff
@@ -1,28 +1,41 @@
-#!/usr/bin/env bash
+#!/usr/bin/env sh
 
 if [ "$PHP_FPM_USER_ID" != "" ]; then
-    (>&2 echo "Warning: PHP_FPM_USER_ID is deprecated. Please use WWW_DATA_USER_ID instead.")
-    WWW_DATA_USER_ID=$PHP_FPM_USER_ID
+    usermod -u $PHP_FPM_USER_ID nobody
```

Comment:
Why did you change from modifying `www-data` to modifying `nobody`?

Reply:
reverted.